server/src/game/Room.ts

```
@@ -118,22 +118,38 @@ class Room {
+
+    const sanitizedMessage = DOMPurify.sanitize(message);
+    const senderName = universal.getNameFromConnectionID(sender.connectionID);
+    const nameColor = sender.playerRank?.color;
+    const userID = sender.ownerUserID ?? null;
     this.chatMessages.push({
       message: sanitizedMessage,
-      sender: sender
+      sender: senderName
     });
     // send to all sockets
     for (let connectionID of this.memberConnectionIDs) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'connectionID' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)